config/set/nette-30-composer.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+declare(strict_types=1);
+
+use Rector\Composer\Rector\ChangePackageVersionRector;
+use Rector\Composer\Rector\RemovePackageRector;
+use Rector\Composer\Rector\ReplacePackageAndVersionRector;
+use Rector\Composer\ValueObject\PackageAndVersion;
+use Rector\Composer\ValueObject\ReplacePackageAndVersion;
+use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
+use Symplify\SymfonyPhpConfig\ValueObjectInliner;
+
+return static function (ContainerConfigurator $containerConfigurator): void {
+    $services = $containerConfigurator->services();
+
+    $services->set(ChangePackageVersionRector::class)
```

Comment:
This was something I didn’t want to do because somewhere it my head was warning which said I need to know exact order of modifications (AddPackage -> RemovePackage has different output like RemovePackage ->AddPackage if we are changing same package). But maybe there is no real world use case for it

Reply:
If there is a conflict, it must be resolved on logical level. As PHP rules are.